src/github/middleware.ts

```diff
@@ -77,6 +77,10 @@ export default (
 		enhanceOctokit(context.github);
 		const webhookEvent = extractWebhookEventNameFromContext(context);
 
+		// Metrics for payload size
+		emitWebhookPayloadMetrics(webhookEvent,
+			Buffer.byteLength(JSON.stringify(context.payload), "utf-8"));
```

Comment:
Can we emit this metric before serializing the palyoad again?

We should either do it before the request was serialised or use req. parameters:
`req.get("content-length")` or `req.socket.bytesRead` if content-length is missing

Reply:
We dont have access to `req` for webhook events, hence, we have to use the payload from context